mcux/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ zephyr_compile_definitions(${MCUX_CPU})
 # practice, drilling down like this avoids the need for repetitive
 # build scripts for every mcux device.
 zephyr_sources(devices/${MCUX_DEVICE}/fsl_clock.c)
+# fsl_colck use the functions in fsl_common.
```

Comment:
nit: fix typo in comment (fsl_colck -> fsl_clock)

Reply:
fixed